src/Host/Client/Impl/Host/LocalRHostConnector.cs

```diff
@@ -93,6 +93,8 @@ public LocalRHostConnector(string name, string rHome, string rhostDirectory = nu
                         FileName = rhostBrokerExe,
                         UseShellExecute = false,
                         Arguments =
+                            $" --logging:logHostOutput true" +
```

Comment:
Logging host output always is a good idea for local broker, because it's the only way to figure out what went wrong with the host if it crashed/terminated before it had a chance to write its own log.

Logging packets can be made configurable, but given that it's not configurable for our Host and Host.Client logs right now, it's probably not worth it unless those become configurable also.

For the time being, I think we'll want logging anyway to help diagnose issues. We can revisit later to decide how (if at all) this needs to be dialed down, and whether any of this needs to be user-configured.